<?xml version="1.0"?>

<launch>

  <arg name="recipe" value="$(find task_planner_dispatcher)/plans/example.yaml" />
  <param name="recipe" value="$(arg recipe)" />

  <node pkg="task_planner_dispatcher" type="dispatcher_single" name="dispatcher_single" output="screen" >
    <param name="topic_request_from_planner" value="/sharework/test/stiima/motion" />
    <param name="topic_feedback_to_planner" value="/sharework/test/stiima/motion/feedback" />
  </node>

</launch>
